### PR TITLE
Add cart page

### PR DESCRIPTION
**Pull request by yash9263**

### CODE REVIEW

**Review by iambizan — APPROVED** [review]

File: styles/pages/cart.css [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+  gap: 8px;
+  margin-left: 12px;
+}
+
+.product-quantity input {
+  max-width: 48px;
+  padding: 0 8px;
+  background: inherit;
+  border: 1px solid;
+  outline: 0;
```
Comment (gauravsinhaweb) [inline styles/pages/cart.css:113]: give outline some value for accessibility.

File: styles/pages/cart.css [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+  padding: 0px 32px 24px;
+}
+
+.cart-heading {
+  text-align: center;
+  margin-bottom: 32px;
+}
+
+.cart-wrapper {
+  display: grid;
```
Comment (manojsarna) [inline styles/pages/cart.css:13]: you could have used px values!!

File: styles/pages/cart.css [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+  grid-template-columns: 1.75fr 1.25fr;
+  gap: 24px;
+}
```
Comment (manojsarna) [inline styles/pages/cart.css:16]: check here too!!

File: styles/pages/cart.css [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+
+@media screen and (max-width: 48rem) {
+  .cart-main {
+    padding: 0px 16px 24px;
+  }
+  .cart-wrapper {
+    grid-template-columns: 1fr;
```
Comment (manojsarna) [inline styles/pages/cart.css:23]: here also

**Review by gauravsinhaweb — APPROVED** [review]
UI looks good, the shopping card in the card could have some distinguish border. Good to merge after changes.

File: styles/pages/cart.css [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+
+.cart-img-wrapper {
+  max-width: 136px;
+  margin-right: 24px;
+}
+
+.cart-img-wrapper img {
+  aspect-ratio: 10 / 14;
+}
+
```
Comment (manojsarna) [inline styles/pages/cart.css:44]: used px values!! please check

File: styles/pages/cart.css [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+  flex-direction: column;
+  flex: 1;
+  padding: 8px;
+}
+
+.product-info h3 {
+  margin-top: 4px;
+  margin-bottom: 4px;
+  margin-right: 32px;
+}
```
Comment (manojsarna) [inline styles/pages/cart.css:56]: here too

File: styles/pages/cart.css [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+.product-desc {
+  margin-top: 16px;
+  margin-right: 16px;
+  font-size: 0.9rem;
```
Comment (manojsarna) [inline styles/pages/cart.css:60]: check for px values

File: styles/pages/cart.css [hunk 8]
```diff
@@ -0,0 +1,169 @@
...
+  padding: 0;
+}
+
+.product-quantity {
+  display: flex;
+  gap: 8px;
+  margin-left: 12px;
+}
+
+.product-quantity input {
```
Comment (manojsarna) [inline styles/pages/cart.css:108]: can you elaborate on why so much px values in the css!!

**Review by manojsarna — CHANGES_REQUESTED** [review]
Overall Looks Fine

Do check the comments

File: styles/pages/cart.css (hunk 8 above)
Comment (yash9263) [reply styles/pages/cart.css:108]: cause I don't want the padding and margin to increase as the font-size increases that's why

File: styles/pages/cart.css (hunk 1 above)
Comment (yash9263) [reply styles/pages/cart.css:113]: yep will do that

**Review by manojsarna — APPROVED** [review]
As per the usage of your code.

Seems Superb to me

UI is fantabulous , Yash

you can merge it :)

File: styles/pages/cart.css [hunk 9]
```diff
@@ -0,0 +1,169 @@
...
+  font-weight: var(--font-weight-bold);
+  margin-top: 24px;
+  border-top: 1.5px solid;
+  padding-top: 16px;
+  margin-bottom: 16px;
+}
+
+.checkout-btn {
+  width: 100%;
+  padding: 12px;
```
Comment (varun13169) [inline styles/pages/cart.css:167]: usage of px

**Review by varun13169 — APPROVED** [review]
Nit Comments, can be addressed as a follow-up

**Pull request merged**